tests/core/tests/authentication.py

```diff
@@ -124,17 +114,9 @@ def test_check_active_true(self):
         bob_doe.set_password('pass')
         bob_doe.save()
         request.META['HTTP_AUTHORIZATION'] = 'Basic %s' % base64.b64encode('bobdoe:pass'.encode('utf-8')).decode('utf-8')
-        self.assertFalse(auth.is_authenticated(request))
-
-    def test_check_active_false(self):
```

Comment:
We might be able to do a @skipIf() here if we really wanted to support non-active user auth (which would only work in Django < 1.10).  I suggest we instead break from the past and remove the require_active flag from the Authentication class entirely; if a user wants to allow auth for a non-active user, they can do so explicitly by creating their own Authentication subclass.